src/main/java/org/jenkinsci/plugins/kubernetes/credentials/OpenShiftBearerTokenCredentialImpl.java

```diff
@@ -138,6 +141,14 @@ private synchronized Token refreshToken(String oauthServerURL, String caCertData
         return extractTokenFromLocation(location.getValue());
     }
 
+    private String getOauthServerUrl(String apiServerURL, String caCertData, boolean skipTLSVerify) throws URISyntaxException, IOException, HttpClientWithTLSOptionsFactory.TLSConfigurationError {
+        URI uri = new URI(apiServerURL);
+        final HttpClientBuilder builder = HttpClientWithTLSOptionsFactory.getBuilder(uri, caCertData, skipTLSVerify);
+        HttpGet discover = new HttpGet(apiServerURL + "/.well-known/oauth-authorization-server");
+        final CloseableHttpResponse response = builder.build().execute(discover);
+        return new JSONObject(EntityUtils.toString(response.getEntity())).getString("authorization_endpoint");
```

Comment:
Replace `org.json` with `net.sf.json` which is provided by the Jenkins ecosystem

```suggestion
        return JSONObject.fromObject(EntityUtils.toString(response.getEntity())).getString("authorization_endpoint");
```

Reply:
added a new commit , removed org.json.JSONObject from pom.xml and replace JSSONObject with the suggested code